aiogram/api/types/base.py

```diff
@@ -15,6 +21,19 @@ class Config:
         allow_population_by_field_name = True
         json_encoders = {datetime.datetime: lambda dt: int(dt.timestamp())}
 
+    @property
+    def bot(self) -> Bot:
```

Comment:
This property is not more needed in due to new design of API methods. Read my comments below.

Reply:
What about `bot.parse_mode` in `User.get_mention` method?